lms/templates/register-form.html

```diff
@@ -104,6 +104,12 @@ <h2 class="sr">${_('Required Information')}</h2>
       <label for="password">${_('Password')}</label>
       <input id="password" type="password" name="password" value="" required aria-required="true" />
     </li>
+    % if settings.REGISTRATION_EXTRA_FIELDS['password_copy'] == 'required':
+    <li class="field ${settings.REGISTRATION_EXTRA_FIELDS['password_copy']} text" id="field-password_copy">
+      <label for="city">${_('Retype password')}</label>
+      <input id="password_copy" type="password" name="password_copy" value="" required aria-required="true" />
```

Comment:
FYI, the ```aria-required``` attribute here (and on line 105 above) is unnecessary.  The ```required``` attribute is sufficient for all users. ```aria-required``` is only necessary when building a custom component and enforcing the required state of it via scripting.  ```aria-required``` would be necessary in this case to inform the accessibiltiy API of this requirement so it could report it to the SR user.  It's not doing any harm in this case, but its generally recommended *not* to have redundant attributes.  This becomes more of a problem when attribute values are toggled, which is not the case here.